applications/plugins/SofaCUDA/sofa/gpu/cuda/CudaDiagonalMass.cpp

```diff
@@ -24,6 +24,11 @@
 #include <sofa/core/behavior/Mass.inl>
 #include <sofa/core/behavior/ForceField.inl>
 #include <sofa/core/ObjectFactory.h>
+#include <SofaBaseTopology/EdgeSetGeometryAlgorithms.inl>
```

Comment:
Include .inl or .h . I'm not expert, what is the best practice?

Reply:
The inl is needed for the implementation of the templated functions, as we instanciate classes with float (and those has been removed from the core of Sofa)

Reply:
The inl are needed each time your have new template instantiations (in this case the float ones).